subsys/bluetooth/services/fast_pair/fp_storage/fp_storage_ak.c

```diff
@@ -473,6 +1037,14 @@ static int fp_storage_ak_reset(void)
 {
 	int err;
 	bool was_enabled = is_enabled;
+	const struct fp_storage_ak_bond_bt_request_cb *registered_bt_request_cb = bt_request_cb;
+
+	if (was_enabled) {
+		err = fp_storage_ak_uninit();
```

Comment:
You can consider merging the independent improvements in a separate PR (like this one or fp_auth.h doc improvement) that could be potentially merged within one day.